deepmerge/strategy/list.py

```diff
@@ -22,3 +22,7 @@ def strategy_prepend(config, path, base, nxt):
     def strategy_append(config, path, base, nxt):
         """ append nxt to base. """
         return base + nxt
+
+    @staticmethod
+    def strategy_append_unduplicated(config, path, base, nxt):
```

Comment:
would you mind adding a docstring? even if it's small, something to clarify unduplicated would be great.